app/helpers/pagination_helper.rb

```diff
@@ -210,6 +211,14 @@ def allowed_params
       @options[:allowed_params] # rubocop:disable Rails/HelperInstanceVariable
     end
 
+    # Customize the Turbo visit action for pagination links. Can be set to "advance" or "replace".
+    #  "advance" - push a new entry onto the history stack.
```

Comment:
❓ is one of those the default action? Or is there a third option, how turbo behaves (when the `data-turbo-action` is unset)?

If so, I'd miss that in the comment to get the understanding.

Reply:
https://github.com/opf/openproject/pull/16793/commits/998eb22dd3ee3b5db016c00c0c6a317272558925